### ISSUE TITLE
implementation not running

### ISSUE THREAD

**Issue by bhowmick1993** [tomguluson92/StyleGAN_PyTorch1.0.1#5]
Hello,
First of all thank you for the code in pytorch. I am currently trying to run your implementation without any change except for the dataset. It seems that from the beginning itself the Generator Loss and the Discriminator Loss is saturating  at 0.6 and 1.3 respectively. After a few epochs they are generating nan values. Could you please help me with this.
Thank you

**Comment by tomguluson92** [comment]
Hi bhownmick, thanks for your question, the strategy of training your own dataset can be listed below:

① squeeze `blur2d` mechanism into the code.
② change the original dataset to your own (like `celebA` and something else).
③ use different training strategy: learning rate, update `Generator` once every five rounds when updating `Discriminator` (The new strategy is updated now ~)
